plugins/outputs/kairosdb/kairosdb_http.go

```diff
@@ -0,0 +1,181 @@
+package kairosdb
```

Comment:
This seems a lot of code for sending some message over HTTP. Please check the `loki` output plugin for a good example of sending metrics over HTTP. If you need more, please let me know why as at a first glance KairoDB HTTP API is pretty straight-forward.